backend/src/models/services/entry/entry.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  Config,
+  Entry,
+  EntryAnalysis,
+  EntryConversation,
+} from '../../index.js';
+import CdGpt from '../../../assistants/gpts/CdGpt.js';
+import { EntryAnalysisType } from '../../entry/entryAnalysis.js';
+import { EntryConversationType } from '../../entry/entryConversation.js';
+import { EntryType } from '../../entry/entry.js';
+import { HydratedDocument } from 'mongoose';
+
+
+/**
+ * Returns array of all entries in a journal
```

Comment:
Gotcha. I think it would be helpful to have a TSDoc linter plugin to enforce styling. Let's make an issue for it and run it towards the end of the ts migration. How's that sound?